packages/orbit-components/transforms/transforms-v23.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * Creates an intl.formatMessage expression container for loading labels
+ *
+ * @param {Object} j - The jscodeshift API object.
+ * @param {string} messageId - The ID of the message to be formatted.
+ * @param {string} defaultMessage - The default message to be displayed if the message ID is not found.
+ */
+function createIntlMessageContainer(j, messageId, defaultMessage) {
+  return j.jsxExpressionContainer(
+    j.callExpression(j.memberExpression(j.identifier("intl"), j.identifier("formatMessage")), [
+      j.objectExpression([
+        j.objectProperty(j.identifier("id"), j.literal(messageId)),
+        j.objectProperty(j.identifier("defaultMessage"), j.literal(defaultMessage)),
+      ]),
+    ]),
+  );
+}
+
+/**
+ * Transforms the given file to replace ariaLabelledBy to ariaLabelledby, and add loading labels to Skeleton components.
+ *
+ * @param {Object} file - The file object containing the source code.
+ * @param {Object} api - The jscodeshift API object.
+ */
+export default function transformer(file, api) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  // Find all Stepper or StepperStateless components
+  root
+    .find(j.JSXElement)
+    .filter(path => {
+      const { openingElement } = path.node;
+
+      // Combine conditions for more efficient filtering
+      return openingElement.name.type === "JSXIdentifier";
```

Comment:
Wouldn't it be more efficient to check for component name during this step/loop? 🤔

Reply:
I think it could be a possible solution, but in my opinion, the return would be long.  Similar solution was done here https://github.com/kiwicom/orbit/commit/af54aea702a16ba528b7fa500210ab7a3303f42f . I think it's not wrong to keep it as it is.